examples/light-switch-app/silabs/include/LightSwitchMgr.h

```diff
@@ -71,7 +71,7 @@ class LightSwitchMgr
      *
      * @param aEvent button event being processed
      */
-    static void SwitchActionEventHandler(AppEvent * aEvent);
+    static void SwitchActionEventHandler(uint16_t eventType);
```

Comment:
what is the benefit of the chaing the function signature? This change seems unnecessary.